### ISSUE TITLE
Allow questions to be un-checked again once they have been checked

### ISSUE THREAD

**Issue by dannylammerhirt** [okfn/opendatasurvey#946]
Currently reviewer are not able to uncheck the data characteristics (the tick-boxes). The entry page will tell them "this question is required" and will force them to tick at least one of the characteristics. This is problematic, if reviewers find that no data characteristics are online and want to correct the submission.

**Comment by brew** [comment]
The question IS required though. If no characteristics are present, why isn't the submission being rejected?

**Comment by dannylammerhirt** [comment]
True. To make it clearer how to answer we could prompt reviewers to check question B2 again. Sth. along the lines "No characteristics found online? Please answer B2 and B2.1 with No." 

This issue came up only by one reviewer. I'd propose to save this issue as a user story for the next GODI edition. WDYT?

**Comment by brew** [comment]
Sounds good.
